types/mapcmd/map.go

```diff
@@ -57,10 +78,122 @@ func New(config *hazelcast.Config) *cobra.Command {
 			return nil
 		},
 	}
-	cmd.AddCommand(NewGet(config), NewPut(config), NewUse())
+	cmd.AddCommand(
+		NewPut(config),
+		NewPutAll(config),
+		NewGet(config),
+		NewGetAll(config),
+		NewRemove(config),
+		NewClear(config),
+		NewUse())
 	return cmd
 }
 
+func printValueBasedOnType(cmd *cobra.Command, value interface{}) {
+	var err error
+	switch v := value.(type) {
+	case serialization.JSON:
+		if err = quick.Highlight(cmd.OutOrStdout(), fmt.Sprintln(v.String()),
```

Comment:
Doesn't `fmt.Sprintln` automatically call `v.String()`?

Reply:
It calls, fixed. [7a991c2](https://github.com/hazelcast/hazelcast-commandline-client/pull/67/commits/7a991c20b78270f67c8e763f4727f6ef1e3a3463)